tests/Linter/ProcessLinterTest.php

```diff
@@ -62,10 +62,12 @@ public function testPrepareCommandOnPhpOnLinux($executable, $file, $expected)
      */
     public function testPrepareCommandOnPhpOnWindows($executable, $file, $expected)
     {
-        $this->assertSame(
-            $expected,
-            AccessibleObject::create(new ProcessLinter($executable))->prepareProcess($file)->getCommandLine()
-        );
+        if (self::hackyPleaseRemoveIsHostUsingWindows()) {
```

Comment:
I can add a test for Mac.

The currently added code *does* fix OS detection? But I imagine it'll be done properly in phpunit at some point?